paddle/go/cclient/cclient.go

```diff
@@ -83,29 +84,46 @@ func cArrayToSlice(p unsafe.Pointer, len int) []byte {
 	return (*[1 << 30]byte)(p)[:len:len]
 }
 
+type selector bool
+
+func (s selector) Select() bool {
+	return bool(s)
+}
+
+type lister []pserver.Server
+
+func (l lister) List() []pserver.Server {
+	return l
```

Comment:
I will write a module who implements this interface that talks to etcd.
However, this PR only includes a module with static pserver addresses. To be used for MPI cluster.
I will send a separate PR for the etcd module.